exercises/queen-attack/queen_attack_test.rb

```diff
@@ -3,152 +3,90 @@
 require 'minitest/autorun'
 require_relative 'queen_attack'
 
-class QueensTest < Minitest::Test
-  def test_default_positions
-    queens = Queens.new
-    assert_equal [0, 3], queens.white
-    assert_equal [7, 3], queens.black
+# Test data version:
+# 82eb00d
+class QueenTest < Minitest::Test
```

Comment:
Looks like there is no space between tests in this file which could make it hard for the end user to read.